docs/UserGuide.md

```diff
@@ -44,65 +44,70 @@ This user guide provides in-depth descriptions of all the available features in
+| Component                              | Use                                                                  |
+|----------------------------------------|----------------------------------------------------------------------|
+| **Command Box**                        | This is where you **enter commands**.                                |
+| **Result Display**                     | This is where the detailed **results** of the command will be shown. |
+| **Guest Card**                         | This is where all **guest information** of a guest will be shown.    |
+| **Status Bar**                         | This shows the location of your **save file**.                       |
+| ![Phone](images/phone.png)             | This is the **phone number** of the guest.                           |
+| ![Email](images/email.png)             | This is the **email address** of the guest.                          |
+| ![Room](images/room.png)               | This is the **room** the guest is staying in.                        |
+| ![DateRange](images/dateRange.png)     | These are the **dates of stay** of the guest.                        |
+| ![NumberOfGuests](images/group.png)    | This is the total **number of guests** in the guest's room.          |
+| ![IsRoomClean](images/isRoomClean.png) | This shows whether the guest's **room has been cleaned**.            |
+| ![Bill](images/bill.png)               | This is the current **bill** of the guest.                           |
+| ![Request](images/request.png)         | This shows any **requests** made by the guest.                       |
 
 [Return to Table of Contents](#table-of-contents)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have `Java 11` or above installed in your Computer.
 
 2. Download the latest `guestbook.jar` from [here](https://github.com/AY2223S1-CS2103T-W16-1/tp/releases). <br> <br>
    ![Ui](images/DownloadInstruction.png) <br><br>
-3. Copy the file to the folder you want to use as the _home folder_ for your GuestBook. <br> <br>
+
+3. Copy the file to the folder you want to use as the **home folder** for your GuestBook. <br> <br>
    ![Ui](images/GuestBookIcon.png) <br><br>
+
 4. Double-click on `guestbook.jar` to start the app. The GUI, similar to the image below, should pop up in a few seconds. 
-Note that the app contains sample data.<br><br>
+Note that the app contains **sample data**.<br><br>
    ![Ui](images/Ui.png) <br><br>
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press _Enter_ to execute it. E.g., typing **`help`** and pressing _Enter_ will open the help window.<br>
```

Comment:
I am unsure which kind of texts are bolded and which are italicised, can you share more?

Reply:
Italics are used for special words while bold is used for important words